deployer/src/config_creator.py

```diff
@@ -68,7 +72,8 @@ def to_gitbook_config(config):
 
 def to_pkgdown_config(config, urls=None):
     if urls:
-        config["sitemap_urls"] = [extract_root_from_input(urls[0]) + "sitemap.xml"]
+        config["sitemap_urls"] = [
```

Comment:
According to the [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation), the closing statement should be on the next line, like the following:
```python
config["sitemap_urls"] = [
    extract_root_from_input(urls[0]) + "sitemap.xml"
]
```
I tend to agree that it makes this kind of statement more readable.
Same as for the other parts of the code.